setup-pg_hba.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+source /env-data.sh
+
+# This script will setup pg_hba.conf
+
+# Reconfigure pg_hba if environment settings changed
+cat $ROOT_CONF/pg_hba.conf.template > $ROOT_CONF/pg_hba.conf
+
+# Custom IP range via docker run -e (https://docs.docker.com/engine/reference/run/#env-environment-variables)
+# Usage is: docker run [...] -e ALLOW_IP_RANGE='192.168.0.0/16'
+if [ "$ALLOW_IP_RANGE" ]
+then
+	echo "Add rule to pg_hba: $ALLOW_IP_RANGE"
+ 	echo "hostssl    all             all             $ALLOW_IP_RANGE              md5" >> $ROOT_CONF/pg_hba.conf
```

Comment:
Use hostssl to allow ssl connection

Reply:
I think this does not 'allow' ssl, it forces it. We should make it so that local (in the docker-compose context) connections do not need to work through ssl since it is much slower. But for the 'wide world' connects (including replication) forcing SSL will be good.